circleguard/investigations.py

```diff
@@ -19,7 +19,7 @@ class Investigations:
     VERSION_SLIDERBUG_FIXED_CUTTING_EDGE = GameVersion(20190111, concrete=True)
 
     @staticmethod
-    def ur(replay, beatmap):
+    def ur(replay, beatmap, adjusted=False):
```

Comment:
it's good that you changed this function, but your work isn't done yet: consumers don't actually call `Investigator.ur`, so this change is useless right now. You need to modify `Circleguard.ur` to take an `adjusted` parameter as well.